packages/hdwallet-core/src/binance.ts

```diff
@@ -0,0 +1,84 @@
+import {BIP32Path} from './wallet'
+
+export interface BinanceGetAddress {
+  addressNList: BIP32Path,
+  showDisplay?: boolean,
+  /** Optional. Required for showDisplay == true. */
+  address?: string,
+}
+
+namespace Binance {
+  namespace sdk {
+
+    export type Coins = Coin[]
+
+    export interface Coin {
+      denom: string
+      amount: string
+    }
+  }
+
+  export interface StdFee {
+    amount: sdk.Coins
+    gas: string
+  }
+
+  export interface StdSignature {
+    pub_key: string
+    signature: string
+  }
+}
+
+export interface BinanceTx {
+  account_number: string
+  chain_id: string
+  data: string
+  memo: string
+  msgs: any
+  signatures?: {
+    pub_key: string
+    signature: string
+  }
+}
+
+export interface BinanceSignTx {
+  addressNList: BIP32Path
+  tx: BinanceTx
+  chain_id: string
+  account_number: string
+  sequence: string
+}
+
+
+export type BinanceSignedTx = BinanceTx
```

Comment:
If signatures are only used for `BinanceSignedTx` maybe remove from `BinanceTx` and use an intersection to create the signed tx interface?
```
export interface BinanceSignedTx = BinanceTx && {
  signatures: {
    pub_key: string
    signature: string
  }
}
```

Reply:
:thinking: